thirdparty/OpenXR/Include/openxr/fb_composition_layer_depth_test.h

```diff
@@ -0,0 +1,83 @@
+// (c) Meta Platforms, Inc. and affiliates. Confidential and proprietary.
```

Comment:
Third-party code should be listed in the:

- https://github.com/godotengine/godot/blob/master/thirdparty/README.md
- https://github.com/godotengine/godot/blob/master/COPYRIGHT.txt

> (c) Meta Platforms, Inc. and affiliates. Confidential and proprietary.

Are these files a part of publicly available and open-source SDK? If it's not the case, they can't be included, you should add a build flag to specify a path to the proprietary SDK instead (and all code that depends on it should be optional and disabled by default).

Reply:
cc @kisg

Reply:
@bruzvg: Sure, we will remove these files and add an external dependency to the Oculus SDK for the extension headers that are required.